src/components/ZoomButton.jsx

```diff
@@ -16,12 +16,12 @@ class ZoomButton extends Component {
                         aria-label="vertical contained primary button group"
                         variant="contained"
                     >
-                        <Button variant="contained" color="secondary" style={{minWidth: "50px", minHeight: "50px", fontSize: "20px"}} active>
-                            +
+                        <Button variant="contained" color="secondary" style={{minWidth: "50px", minHeight: "50px", fontSize: "20px"}} onClick={this.props.onZoomIn} active>
+                            <b>+</b>
```

Comment:
Do we have asset for those two buttons? I think use image / svg is preferred over character